.github/workflows/windows-x64.yaml

```diff
@@ -0,0 +1,47 @@
+name: windows-x64
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Could you please add another branch here, like `alpha` or `preflight`? Or we'll be unable to sensibly debug it.

Reply:
Ok.

By the way,
https://github.com/kkm000/openfst/blob/3ac85e463b90eba042d33b2d54ceb75d5aeb1144/.github/workflows/windows-x64.yaml#L12

the above line means you can trigger any branch manually, even if those branches are not listed here.

Reply:
Ah, if a manual run can be triggered, then it's not a concern at all!